tsec-libsodium/src/main/scala/tsec/libsodium/ScalaSodium.scala

```diff
@@ -39,23 +42,31 @@ object ScalaSodium
   private val libraryName = "sodiumjni"
 
   /** Check the sodium version is compatible with the api's we're using **/
-  private def checkVersion(sodiumString: String): Unit =
+  private def checkVersion(sodiumString: String): Unit = {
+    val error = new SodiumLoadError(s"Unsupported libsodium version $sodiumString. Please upgrade to version 1.0.12+")
+
     sodiumString.trim.split("\\.") match {
       case Array(major, med, minor) =>
         if (major.toInt < 1 || (minor.toInt < 12 && med.toInt == 0)) {
-          throw new SodiumLoadError(s"Unsupported libsodium version $sodiumString. Please upgrade to version 1.0.12+")
+          logger.error(error)("Unsupported version")
+          throw error
         } else
-          println("\n[tsec] Loading libsodium jni... Hold tight Asznee\n")
+          logger.info("Loading libsodium jni... Hold tight Asznee")
       case _ =>
-        throw new SodiumLoadError(s"Unsupported libsodium version $sodiumString. Please upgrade to version 1.0.12+")
+        logger.error(error)("Unsupported version")
+        throw error
     }
+  }
 
   private[tsec] lazy val Sodium: ScalaSodium = {
     System.loadLibrary(libraryName)
     val sodium = new ScalaSodium()
     checkVersion(SodiumJNI.sodium_version_string.asInstanceOf[String])
-    if (sodium.sodium_init < 0)
-      throw new SodiumLoadError("Has not been initialized properly")
+    if (sodium.sodium_init < 0) {
+      val err = new SodiumLoadError("Has not been initialized properly")
+      logger.error(err)("ScalaSodium has not been initialized properly")
+      throw err
+    }
```

Comment:
What do you think of a `logger.info("Scalasodium... on!")` or whatever funny message you want indicating libsodium was loaded properly here?